tpcx_bb/xbb_tools/q27_get_review_sentence_utils.py

```diff
@@ -0,0 +1,353 @@
+#
+# Copyright (c) 2019-2020, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+import cudf
+import torch
+from numba import cuda
+from .q27_bert_utils import get_stride
+import cupy as cp
+import logging
+import numpy as np
+
+
+def get_review_sentence(
+    tokenized_d, predicted_label_t, vocab2id, id2vocab, org_labels=[2, 5]
+):
+    """
+        Given a tokenized_d and predicted_label_t
+        We return the  sentences that contain labels for either 5 or 2
+
+        #### Detailed Workflow ####
+
+        ## This function contains the review sentence gathering part of workflow
+        # Given a tokenized_d and predicted_label_t
+        # we gather the  sentences that contain labels for it
+
+        ### The workflow is as follows:
```

Comment:
This is the main function of this file, all other functions here are to support this function.